### PR TITLE
Add schemas for MusicAlbum and Genre

### PR DESCRIPTION
**Pull request by bienvenuushindi**
- [x] #13

### CODE REVIEW

**Review by pinkmoon25 — APPROVED** [review]
## APPROVED 🟢
Hi @bienvenuushindi 🙏 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights
No linter errors. ✔️
GitHub flow is followed. ✔️
PR has a good title and summary. ✔️
All the schemas are well implemented. ✔️
Cheers and Happy coding! 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Review by ChrispaixK — APPROVED** [review]
# Status : APPROVED 🟢
Hi @bienvenuushindi , well done 👍🏾 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights
GitHub flow is followed. ✔️
PR has a good title and summary. ✔️
queries well implemented. ✔️
Cheers and Happy coding! 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: schemas/schemas.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+CREATE TABLE music_album
+(
+    id         INT GENERATED ALWAYS AS IDENTITY,
+    on_spotify BOOLEAN,
+    CONSTRAINT fk_item
+        FOREIGN KEY (item_id)
+            REFERENCES item (id)
+            ON DELETE CASCADE,
+    PRIMARY KEY (id)
```
Comment (pinkmoon25) [inline schemas/schemas.sql:33]: hi @bienvenuushindi , I think you forgot to add an Item_id column here.

Comment (bienvenuushindi) [reply schemas/schemas.sql:33]: @PinkMoon25  Thank you very much

**Pull request merged**